collectors/channels_collector.go

```diff
@@ -101,6 +102,11 @@ func NewChannelsCollector(lnd lndclient.LightningClient, errChan chan<- error,
 			"total number of inactive channels",
 			[]string{"state"}, nil,
 		),
+		numClosedChannels: prometheus.NewDesc(
+			"lnd_closed_channels_total",
+			"total number of closed channels",
+			[]string{"type"}, nil,
```

Comment:
`type` -> `close_type`? bit too generic of a label for me, since we technically have channel types as well.

Reply:
done